CHAGNELOG.md

```diff
@@ -1,6 +1,21 @@
 # Changelog
 ## [Unreleased]
 
+## [0.10.3]
+
+### Added
+- [#115](https://github.com/cloud-hypervisor/fuse-backend-rs/pull/115): transport: pre-allocate VecDeque to avoid expending at runtime
+- [#119](https://github.com/cloud-hypervisor/fuse-backend-rs/pull/119): Support non-privileged users
```

Comment:
These three commits should be in "Added" as well?
![image](https://user-images.githubusercontent.com/45746212/233036536-df96ad3a-d2e8-472b-aafa-8c8701e04340.png)

Reply:
Good point! Added these. Thanks!